.gitignore

```diff
@@ -20,3 +20,10 @@ classes
 obj
 
 .DS_Store
+
+# Gradle
+.gradle
+jniLibs
+build
+local.properties
+reports
```

Comment:
nit: trailing newline.

also, delete the Maven ignores ("target", "pom.xml.*", "release.properties")